crates/musig2/src/rkyv_wrappers.rs

```diff
@@ -0,0 +1,190 @@
+use rkyv::{Archive, Deserialize, Serialize};
+use secp256k1::ffi::CPtr;
```

Comment:
Just do the same to what we did on `strata` for the `musig2` stuff for Borsh.
See https://github.com/alpenlabs/strata/blob/a0d413d711bb3af5ce312544f45600806f3f18bb/crates/primitives/src/bridge.rs#L225-L263